specification/communication/Communication.Messages/client.tsp

```diff
@@ -16,8 +16,8 @@ namespace ClientForAcsMessages;
 })
 interface NotificationMessagesClient extends AzureCommunicationMessagesService.NotificationMessagesOperations
 {
-  @access(Access.internal)
-  getMedia is AzureCommunicationMessagesService.StreamOperations.getStream;
+  @access(Access.public)
```

Comment:
Do JS and JAVA not require a Stream as return type? I needed to make this internal and have a wrapper method in .NET because the generated method spits out BinaryData while we want Stream on the SDK.

Reply:
I didn't knew we need to add wrapper. What kind of wrapper you add. I been doing same with swagger for Java.

[   JS => {](https://github.com/Azure/azure-sdk-for-js-pr/blob/803535a042400f42ce433d97754facedc518f162/sdk/communication/communication-messages-rest/samples-dev/DownloadMedia.ts#L18C25-L18C25)

https://github.com/Azure/azure-sdk-for-java-pr/blob/1b2de8cf9729a94e35cb330dc4dbb55e546f973c/sdk/communication/azure-communication-messages/src/samples/java/com/azure/communication/messages/samples/MediaDownloadSample.java#L27